Replicator/RevFinder.cc

```diff
@@ -217,6 +217,7 @@ namespace litecore::repl {
                 auto mode = (deletion < 4) ? RevocationMode::kRevokedAccess
                                            : RevocationMode::kRemovedFromChannel;
                 revoked.emplace_back(new RevToInsert(docID, mode));
+                sequences.push_back({RemoteSequence(change[0]), 0});
```

Comment:
This is wrong -- the `sequences` array is parallel with `docIDs` and `revIDs`. It isn't used for revoked revs.